components/lwip/port/esp32/include/lwipopts.h

```diff
@@ -269,6 +269,12 @@
    ---------- SNMP options ----------
    ----------------------------------
 */
+#define LWIP_SNMP                       CONFIG_LWIP_SNMP_SUPPORT
+#define MIB2_STATS                      CONFIG_LWIP_STATS
```

Comment:
This macro introduces a define conflict with https://github.com/espressif/esp-lwip/blob/2.1.2-esp/src/include/lwip/opt.h#L2268 for default sdkconfig:
```
In file included from /home/david/dev/esp-idf/components/lwip/lwip/src/include/lwip/ip_addr.h:40,
                 from /home/david/dev/esp-idf/components/tcpip_adapter/include/tcpip_adapter_types.h:18,
                 from /home/david/dev/esp-idf/components/tcpip_adapter/include/tcpip_adapter.h:24,
                 from /home/david/dev/esp-idf/components/esp_netif/include/esp_netif.h:35,
                 from /home/david/dev/esp-idf/components/esp_event/include/esp_event_legacy.h:22,
                 from /home/david/dev/esp-idf/components/esp_event/include/esp_event.h:26,
                 from /home/david/dev/esp-idf/components/esp_wifi/include/esp_wifi.h:64,
                 from /home/david/dev/esp-idf/examples/protocols/sockets/tcp_server/main/tcp_server.c:14:
/home/david/dev/esp-idf/components/lwip/lwip/src/include/lwip/opt.h:2282: warning: "MIB2_STATS" redefined
 #define MIB2_STATS                      0
 
In file included from /home/david/dev/esp-idf/components/lwip/lwip/src/include/lwip/opt.h:51,
                 from /home/david/dev/esp-idf/components/lwip/lwip/src/include/lwip/ip_addr.h:40,
                 from /home/david/dev/esp-idf/components/tcpip_adapter/include/tcpip_adapter_types.h:18,
                 from /home/david/dev/esp-idf/components/tcpip_adapter/include/tcpip_adapter.h:24,
                 from /home/david/dev/esp-idf/components/esp_netif/include/esp_netif.h:35,
                 from /home/david/dev/esp-idf/components/esp_event/include/esp_event_legacy.h:22,
                 from /home/david/dev/esp-idf/components/esp_event/include/esp_event.h:26,
                 from /home/david/dev/esp-idf/components/esp_wifi/include/esp_wifi.h:64,
                 from /home/david/dev/esp-idf/examples/protocols/sockets/tcp_server/main/tcp_server.c:14:
/home/david/dev/esp-idf/components/lwip/port/esp32/include/lwipopts.h:273: note: this is the location of the previous definition
 #define MIB2_STATS                      CONFIG_LWIP_STATS
```